chain/ethereum/src/lib.rs

```diff
@@ -6,6 +6,9 @@ pub mod network_indexer;
 pub mod runtime;
 mod transport;
 
+// StreamingFast components
+pub mod sf;
```

Comment:
Feel free to put your code right into the corresponding crates/modules. No need to namespace it like this from my POV.